Gems/EMotionFX/Code/MCore/Source/AttributeColor.h

```diff
@@ -66,16 +68,29 @@ namespace MCore
             m_value.Set(vec4.GetX(), vec4.GetY(), vec4.GetZ(), vec4.GetW());
             return true;
         }
-        bool ConvertToString(AZStd::string& outString) const override      { AZStd::to_string(outString, AZ::Vector4(m_value.m_r, m_value.m_g, m_value.m_b, m_value.m_a)); return true; }
-        size_t GetClassSize() const override                        { return sizeof(AttributeColor); }
-        AZ::u32 GetDefaultInterfaceType() const override             { return ATTRIBUTE_INTERFACETYPE_COLOR; }
+        
+        bool ConvertToString(AZStd::string& outString) const override
+        {
+            AZStd::to_string(outString, m_value.GetAsVector4());
```

Comment:
update: There is a `to_string` overload for `AZ::Color` (see `MathStringConversions.h`) so we can just use `m_value` directly

Reply:
I just been do this

```
AZStd::to_string(outString, m_value);
```

Reply:
I see that changes this functionally but it can't be too bad right?

Reply:
I don't think so, the rest of the PR changes functionality too anyway 😛